Sources/BrowserServicesKit/Email/EmailKeychainManager.swift

```diff
@@ -68,16 +68,19 @@ extension EmailKeychainManager: EmailManagerStorage {
     }
 
     public func getWaitlistToken() -> String? {
-        EmailKeychainManager.getString(forField: .waitlistToken)
+        return try? EmailKeychainManager.getString(forField: .waitlistToken)
```

Comment:
Waitlist functions are deliberately ignored, these are all going away soon and there's more work required if these are made throwing.